stories/Form/FormHeader/FormHeader.stories.tsx

```diff
@@ -56,3 +56,14 @@ FormHeaderComponent2.args = {
   ),
   buttons: formButtonList.slice(0, 2)
 };
+
+export const FormHeaderComponent3 = Template.bind({});
```

Comment:
```suggestion
export const FormHeaderWithButtons = Template.bind({});
```
or something similar, please make the titles more meaningful